app/actions/CommonActions.scala

```diff
@@ -23,7 +23,7 @@ object CommonActions {
 
   val AuthorisedTester = GoogleAuthenticatedStaffAction andThen requireGroup[GoogleAuthRequest](Set(
     "directteam@guardian.co.uk",
-    "subscriptions.dev@guardian.co.uk",
```

Comment:
Please can you leave that in, and add membership.dev additionally? It was just that you were removed from subscriptions.dev - there might be others of us who are on that and not in membership.dev!